plugins/Goals/tests/System/ProcessDependentArchiveTest.php

```diff
@@ -34,6 +34,12 @@ public function tearDown(): void
         parent::tearDown();
     }
 
+   public function test_numArchivesCreated_day()
+    {
+        API::getInstance()->getMetrics(self::$fixture->idSite, 'day', '2009-01-04');
+        $this->assertNumRangeArchives(5, 1); // days;
```

Comment:
Doesn't this test have the same result on 4.x-dev? Wondering how that should cover the changes though.

Reply:
In 4.x-dev it only generates 3 archives. We debugged and tested together to make sure this tests the right use case.

Reply:
Hm... Seems i didn't checkout the test changes correctly when I tried running that test on 4.x-dev last time. Tried that again right now, and it indeed fails on 4.x-dev...